src/tools/wix/Xsd/wix.xsd

```diff
@@ -1232,6 +1233,43 @@
       </xs:attribute>
     </xs:complexType>
   </xs:element>
+  <xs:element name="CommandLine">
+    <xs:annotation>
+      <xs:documentation>Describes additional, conditional command-line arguments for an ExePackage.</xs:documentation>
+      <xs:appinfo>
+        <xse:parent namespace="http://schemas.microsoft.com/wix/2006/wi" ref="ExePackage" />
```

Comment:
We have hundreds of those...oh well. MsiProperty is in the queue (brain queue, not PR queue).